common/db/tag_store/dynamo.go

```diff
@@ -71,7 +87,7 @@ func (d *DynamoTagStore) SelectByQuery(entityType, entityID string) (models.Tags
 	}
```

Comment:
The switch statement reads like there are many different ways to query this table. You can create indexes to help simplify some of these, however queries like those on `EntityType` itself looks like they have the potential to return too many results and overrun your table capacity.